app/javascript/vue/BookCard.vue

```diff
@@ -17,6 +17,11 @@
   export default {
     props: {
       book: Object
+    },
+    data () {
+      return {
+        bookLocal: this.book
```

Comment:
@isabloCh I don't see you modify `bookLocal` anywhere in this file. Why do use `bookLocal` instead of `book` in this file then?

Reply:
+